src/ddmd/astnull.d

```diff
@@ -38,47 +38,47 @@ struct ASTNull
     alias PROTexport        = PROTKIND.PROTexport;
     alias PROTundefined     = PROTKIND.PROTundefined;
 
-    static int MODconst            = 0;
```

Comment:
this is not meant to be used.
it's just a template...
so I took the four letter word over 3 long words